docs/docs/index.md

```diff
@@ -1,143 +1,134 @@
+    When creating a _Conda_ virtual environment, a specific Python version may be specified. Create your virtual environment with a Python environment that is avaiable on your Posit Connect server.
 
-    --8<-- "snippets/python_pkg.md"
+    <div class="code-title">Terminal</div>
+    ```shell
+    conda create --name rsconnect-jupyter python=3.8
+    ```
 
-!!! important
-    Be sure to run Jupyter from this virtual environment, not from
-    another installation, or the `rsconnect-jupyter` extension will
-    not be available. To do so, you will need to activate the virtual
-    environment in each new terminal session before you run `jupyter`.
+#### Python Virtual Environment (virtualenv)
 
----
+<div class="code-title">Terminal</div>
+```shell
+python -m pip install virtualenv
+python -m virtualenv .venv
+source .venv/bin/activate
+```
 
-### Installing to Jupyter running on Posit Workbench
+Next, following the [installation guide](#installation).
 
-- If you are installing `rsconnect-jupyter` to Jupyter running on RStudio Server Pro, see
-the [RStudio Server Pro documentation on Jupyter Notebooks](https://docs.rstudio.com/rsp/integration/jupyter-standalone/#4-install-jupyter-notebooks-jupyterlab-and-python-packages)
-for instructions on installing the plugin to the right location.
+!!! Tip
+    Running `source .venv/bin/activate` activates the virtual environment. While the `virtualenv` is active, Python-related commands like `python`, `pip`, and `jupyter` will use to copies located inside the virtual environment.
 
-- Once you complete the installation instructions, please return to this document for additional information such as [Upgrading](upgrading) or [Usage](usage) instructions.
+### Posit Workbench
 
----
+See [Installing Python on Posit Workbench](https://docs.posit.co/rsw/integration/jupyter-standalone/#4-install-jupyter-notebooks-jupyterlab-and-python-packages).
 
-### Installation in JupyterHub
+Once you complete the installation instructions, please return to this document for additional information such as [Upgrading](upgrading) or [Usage](usage) instructions.
 
-In JupyterHub, follow these directions to install the
-`rsconnect-jupyter` package into the Python environment where the Jupyter
-notebook server and kernel are installed:
+## JupyterHub
 
---8<-- "snippets/python_pkg.md"
+Follow the [installation guide](#installation) to install `rsconnect-jupyter` onto the _JupyterHub_ server.
 
-Typically those will be the same
-environment. If you've configured separate kernel environments, install the
-`rsconnect-jupyter` package in the notebook server environment as well as each
-kernel environment.
+If you've configured separate kernel environments, repeat the installation guide for each kernel environment.
 
-The exact install location depends on your JupyterHub configuration.
+!!! Note
+    The exact install location depends on your JupyterHub configuration.
 
-#### JupyterHub Example Configuration
+#### Quick Start Example
```

Comment:
We might want to clarify that this example is to illustrate a minimal working configuration and isn't intended to be a recommendation about how to run JupyterHub.